tools/template/cartesi-machine.template

```diff
@@ -1,5 +1,5 @@
 #!/bin/sh
 export LUA_PATH_5_4="ARG_LUA_PATH;${LUA_PATH_5_4:-;}"
 export LUA_CPATH_5_4="ARG_LUA_CPATH;${LUA_CPATH_5_4:-;}"
-export CARTESI_IMAGES_PATH="ARG_INSTALL_PATH"
+export CARTESI_IMAGES_PATH="${CARTESI_IMAGES_PATH:-ARG_IMAGES_PATH}"
```

Comment:
To be more consistent, the same change could be done in `cartesi-machine-tests.template`.

Reply:
cartesi-machine-tests doesn't use images. I will remove it there. Maybe add the tests path as it should. Spotted an old bug there. :)